packages/core/src/base/staticClient.ts

```diff
@@ -78,6 +79,25 @@ export function getHeader(
   return header;
 }
 
+/**
+ * Format the request being made for logging.
+ *
+ * @param resource The resource being requested
+ * @param fetchOptions The options to the fetch call
+ */
+export const formatFetchForInfoLog = 
```

Comment:
[NIT] really minor...but should this be `formatRequestForInfoLog` instead

Reply:
I had thought that, but it felt misleading since there is a Request object and that's not what this is. The input is the arguments to the fetch call including the fetch options. It ends up being more fetch specific than Request specific.